File: final_qualifying_work/Projects/server/Models/Dtos/PersonDto.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace server.Models.Dtos
{
    public class PersonDto
    {
        public uint PersonId { get; set; }
        public string Email { get; set; } = null!;
        public string Phone { get; set; } = null!;
        public string LastName { get; set; } = null!;
        public string FirstName { get; set; } = null!;
        public string? Patronymic { get; set; }
        public DateOnly Birth { get; set; }
        public string? DriveLisense { get; set; }
    }
}

File: final_qualifying_work/Projects/data_provider_app/app/src/main/java/com/example/data_provider_app/dto/PersonDto.kt
package com.example.data_provider_app.dto

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import java.time.LocalDate
import java.time.LocalDateTime

@Parcelize
data class PersonDto(
    val personId: UInt,
    val createdAt: LocalDateTime,
    val email: String,
    val phone: String,
    val lastName: String,
    val firstName: String,
    val patronymic: String?,
    val birth: LocalDate,
    val driveLicense: String?,
    val avatarId: UInt
) : Parcelable
